kong/clustering/rpc/manager.lua

```diff
@@ -188,10 +190,37 @@ function _M:_handle_meta_call(c, cert)
   assert(type(info.kong_hostname) == "string")
   assert(type(info.kong_conf) == "table")
 
+  local version = info.kong_version
```

Comment:
not a blocker: should we call it dp_version for additional clarity?

Reply:
I changed that.